flocker/testtools/__init__.py

```diff
@@ -47,11 +47,15 @@
 from twisted.trial.unittest import TestCase
 from twisted.python.logfile import LogFile
 
+from ._flaky import flaky
 from .. import __version__
 from ..common.script import (
     FlockerScriptRunner, ICommandLineScript)
 
 
+# Export for other places to import. Be still, flake8.
+flaky
+
 # This is currently set to the minimum size for a SATA based Rackspace Cloud
```

Comment:
It doesn't do anything except trick the static checker (flake8) into not emitting a particular warning (that a name is imported but unused).

I think putting the name in `__all__` is a better solution - since this is /also/ how you trick programmers into thinking that the name is being imported just so it can be re-exported. :wink: 

In this case, because of the massive pile of code also in this `__init__`, defining an `__all__` is probably more annoying.  Maybe the massive pile should move, though.  But that doesn't sound like the kind of thing that should block this change.